CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Updating `.notice-banner__content` styles so that they're actually visible [#820](https://github.com/hmrc/assets-frontend/pull/820)
 
+## [2.251.2] - 2017-10-16
+### Added
+- Styling for new full width banner on frontend-template-provider
```

Comment:
This needs to be in the `Unreleased` section and have a link to this PR. It will get a version number when it gets released.